lesson3_tsk8.py

```diff
@@ -0,0 +1,51 @@
+"""
+Матрица 5x4 заполняется вводом с клавиатуры, кроме последних элементов строк.
+Программа должна вычислять сумму введенных элементов каждой строки и записывать ее в последнюю ячейку строки.
+В конце следует вывести полученную матрицу.
+"""
+
+r_count = 1
+last_clm = []
+row_sum = 0
+matrix_1 = []
+matrix_2 = []
+
+
+print('Введите числа матрицы через пробел. Для перехода на новую строку нажмите Enter')
+while r_count <= 4:
+    row = input()
+    matrix_1.append(row.split(' '))
```

Comment:
если оставить split пустым то он будет через пробел по умолчанию.
```suggestion
    matrix_1.append(row.split())
```